draft-aegis-mimi-arch-latest.md

```diff
@@ -287,7 +355,7 @@ HTTP POST /apps/{appId}/clients
 Input:
 
 {
-    clientIdentity: Base64(IdentityProvider.client_identity),
```

Comment:
i think it should be `clientIdentity`. `clientId` is a meaningless identifier created by the service. `clientIdentity` is a handle (using Rohan's terms) that can change, like alice@org.